Grbl_Esp32/ProcessSettings.cpp

```diff
@@ -315,61 +307,226 @@ err_t doJog(const char* value, auth_t auth_level, ESPResponseStream* out) {
-    { STATUS_SOFT_LIMIT_ERROR , "Soft limit error", },
-    { STATUS_OVERFLOW , "Line too long", },
-    { STATUS_MAX_STEP_RATE_EXCEEDED , "Max step rate exceeded", },
-    { STATUS_CHECK_DOOR , "Check door", },
-    { STATUS_LINE_LENGTH_EXCEEDED , "Startup line too long", },
-    { STATUS_TRAVEL_EXCEEDED , "Max travel exceeded during jog", },
-    { STATUS_INVALID_JOG_COMMAND , "Invalid jog command", },
-    { STATUS_SETTING_DISABLED_LASER , "Laser mode requires PWM output", },
-    { STATUS_GCODE_UNSUPPORTED_COMMAND , "Unsupported GCode command", },
-    { STATUS_GCODE_MODAL_GROUP_VIOLATION , "Gcode modal group violation", },
-    { STATUS_GCODE_UNDEFINED_FEED_RATE , "Gcode undefined feed rate", },
-    { STATUS_GCODE_COMMAND_VALUE_NOT_INTEGER , "Gcode command value not integer", },
-    { STATUS_GCODE_AXIS_COMMAND_CONFLICT , "Gcode axis command conflict", },
-    { STATUS_GCODE_WORD_REPEATED , "Gcode word repeated", },
-    { STATUS_GCODE_NO_AXIS_WORDS , "Gcode no axis words", },
-    { STATUS_GCODE_INVALID_LINE_NUMBER , "Gcode invalid line number", },
-    { STATUS_GCODE_VALUE_WORD_MISSING , "Gcode value word missing", },
-    { STATUS_GCODE_UNSUPPORTED_COORD_SYS , "Gcode unsupported coordinate system", },
-    { STATUS_GCODE_G53_INVALID_MOTION_MODE , "Gcode G53 invalid motion mode", },
-    { STATUS_GCODE_AXIS_WORDS_EXIST , "Gcode extra axis words", },
-    { STATUS_GCODE_NO_AXIS_WORDS_IN_PLANE , "Gcode no axis words in plane", },
-    { STATUS_GCODE_INVALID_TARGET , "Gcode invalid target", },
-    { STATUS_GCODE_ARC_RADIUS_ERROR , "Gcode arc radius error", },
-    { STATUS_GCODE_NO_OFFSETS_IN_PLANE , "Gcode no offsets in plane", },
-    { STATUS_GCODE_UNUSED_WORDS , "Gcode unused words", },
-    { STATUS_GCODE_G43_DYNAMIC_AXIS_ERROR , "Gcode G43 dynamic axis error", },
-    { STATUS_GCODE_MAX_VALUE_EXCEEDED , "Gcode max value exceeded", },
-    { STATUS_P_PARAM_MAX_EXCEEDED , "P param max exceeded", },
-    { STATUS_SD_FAILED_MOUNT , "SD failed mount", },
-    { STATUS_SD_FAILED_READ , "SD failed read", },
-    { STATUS_SD_FAILED_OPEN_DIR , "SD failed to open directory", },
-    { STATUS_SD_DIR_NOT_FOUND , "SD directory not found", },
-    { STATUS_SD_FILE_EMPTY , "SD file empty", },
-    { STATUS_SD_FILE_NOT_FOUND , "SD file not found", },
-    { STATUS_SD_FAILED_OPEN_FILE , "SD failed to open file", },
-    { STATUS_SD_FAILED_BUSY , "SD is busy", },
-    { STATUS_SD_FAILED_DEL_DIR, "SD failed to delete directory", },
-    { STATUS_SD_FAILED_DEL_FILE, "SD failed to delete file", },
-    { STATUS_BT_FAIL_BEGIN , "Bluetooth failed to start", },
-    { STATUS_WIFI_FAIL_BEGIN , "WiFi failed to start", },
-    { STATUS_NUMBER_RANGE , "Number out of range for setting", },
-    { STATUS_INVALID_VALUE , "Invalid value for setting", },
-    { STATUS_MESSAGE_FAILED , "Failed to send message", },
-    { STATUS_NVS_SET_FAILED , "Failed to store setting", },
-    { STATUS_AUTHENTICATION_FAILED, "Authentication failed!", },
+    {
```

Comment:
And here it messed up the tabular form in a different way.  Perhaps the trailing comma after the strings has something to do with it?